app/services/claim_fast_tracking/max_rating_annotator.rb

```diff
@@ -13,9 +14,8 @@ def self.annotate_disabilities(rated_disabilities_response)
 
       diagnostic_codes = rated_disabilities_response.rated_disabilities
                                                     .compact # filter out nil entries in rated_disabilities
+                                                    .select { |rd| eligible_for_request?(rd) } # select only eligible
```

Comment:
The `eligible_for_request?` method is more fleshed out, and now takes the entire RatedDisability object, not just the diagnostic code.